src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    public String getRandomColor() {
+        Random random = new Random();
```

Comment:
not fixed
move to the class variables, like 
![image](https://user-images.githubusercontent.com/12864128/230045785-ae0cff7a-51f2-4fdd-b290-a0a4fb03781b.png)